tracer/src/Datadog.Trace/Iast/IastModule.cs

```diff
@@ -616,7 +594,17 @@ public static bool AddRequestVulnerabilitiesAllowed()
         return isRequest && traceContext?.IastRequestContext?.AddVulnerabilitiesAllowed() == true;
     }
 
-    private static IastModuleResponse GetScope(string evidenceValue, IntegrationId integrationId, string vulnerabilityType, string operationName, Func<TaintedObject, bool>? taintValidator = null, bool addLocation = true, int? hash = null, StackTrace? externalStack = null, SecureMarks exclusionSecureMarks = SecureMarks.None)
+    private static IastModuleResponse GetScope(
+        string evidenceValue,
+        IntegrationId integrationId,
+        string vulnerabilityType,
+        string operationName,
+        Func<TaintedObject, bool>? taintValidator = null,
+        bool addLocation = true,
+        int? hash = null,
+        StackTrace? externalStack = null,
+        SecureMarks exclusionSecureMarks = SecureMarks.None,
+        SourceType[]? safeSources = null)
```

Comment:
Could we make safeSources non-null here and remove a bunch of null checks? AFAICT you always pass it in? 🤔

Reply:
There are two cases where this is null (SQLi and XSS)